src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedIntrinsicOperatorSymbol.cs

```diff
@@ -491,6 +491,11 @@ public override ImmutableArray<CustomModifier> RefCustomModifiers
             {
                 get { return ImmutableArray<CustomModifier>.Empty; }
             }
+
+            internal override MarshalPseudoCustomAttributeData MarshallingInformation
```

Comment:
>internal override MarshalPseudoCustomAttributeData MarshallingInformation [](start = 12, length = 73)

It is not clear what motivated this change.

Reply:
Because I deleted the implementation in SynthesizedParameterSymbolBase, it was necessary to provide an implementation in each subtype.

---
In reply to: [373183931](https://github.com/dotnet/roslyn/pull/41299#discussion_r373183931) [](ancestors = 373183931)